topics/introduction/slides/introduction.html

```diff
@@ -19,20 +19,20 @@
 
 [![Galaxy](../../../shared/images/galaxy_logo_25percent_transparent.png)](https://galaxyproject.org/)
 
-- **Web-based** platform for computational biomedical research (analysis and data integration)
-  - Developed at Penn State, Johns Hopkins, OHSU and Cleveland Clinic with substantial outside contributions
-  - **Open source** under [Academic Free License](https://opensource.org/licenses/AFL-3.0)
-- More than 7,500 [citations](https://www.zotero.org/groups/1732893/galaxy)
-- More than 150 [public Galaxy resources](https://galaxyproject.org/use/)
-  - 120+ public servers, many more non-public
-  - Both general-purpose and domain-specific
+**Data Intensive *analysis* for everyone**
+- Versatile and reproducible workflows
+- **Web** platform
+- **Open source** under [Academic Free License](https://opensource.org/licenses/AFL-3.0)
+- Developed at Penn State, Johns Hopkins, OHSU and Cleveland Clinic with substantial outside contributions
 
 ![Galaxy resources](../images/titleless-use-resource-banner.png)
 
 ???
 
 - The Galaxy Team is composed by bioinformaticians and software engineers
 - OHSU = Oregon Health & Science University
+- Customize the number of public resources
```

Comment:
Not sure what this means, can you elaborate?

Reply:
The idea is to explain a workflow by comparing it with cooking recipes.
The reason is that we want to prepare an explanation for users researchers, citizens and hobbyists that are not familiar with data intensive research.

Reply:
Exactly, I often descrive workflow as recipes when giving help.

Eg: All of the "instructions" (tools, parameters), but none of the actual "ingredients" (data).